Makefile

```diff
@@ -33,8 +33,8 @@ install-safety:
 	pip install safety
 
 check-dependencies: install-safety ## Scan dependencies for security vulnerabilities
-	# The ignored issues are documented in requirements-app.txt.
-	safety check -r requirements.txt --full-report -i 40399 -i 40416 -i 42497 -i 42498 -i 43738 -i 47833 -i 51668 -i 52548
+	# Ignored issues, if any, are documented in requirements-app.txt.
+	safety check -r requirements.txt --full-report -i 40416 -i 42497 -i 42498 -i 43738 -i 51668
```

Comment:
The remaining vulnerabilities mostly relate to Celery, which we intend to upgrade in the near future.